buildchain/buildchain/salt_tree.py

```diff
@@ -209,7 +209,7 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
     Path('salt/_modules/containerd.py'),
     Path('salt/_modules/cri.py'),
     Path('salt/_modules/docker_registry.py'),
-    Path('salt/_modules/kubernetes.py'),
+    Path('salt/_modules/kubernetesmod.py'),
```

Comment:
You should explain in the commit message WHY we must rename the file.

Reply:
A link to the Saltstack PR/issue could be enough IMHO

Reply:
Yes an no, I really would like to have some info that are readable offline.